plugins/mcreator-localization/help/default/biome/generate_bastion_remnants.md

```diff
@@ -0,0 +1 @@
+Select this setting to have Bastion remnants in your biome.
```

Comment:
The capitalization in the tooltips isn't consistent with the GUI: Bastion, Buried, Mineshafts, Ruined, Shipwrecks, Swamp should be lowercase